Loosen the requirements when the datagetter is used as a library

R2ZER0 opened the pull request:
by switching setup.py to use requirements.in instead of requirements.txt.

Now that requirements.in specifies compatible ranges, this allows downstream library uses to upgrade to compatible sub-requirement versions, e.g. for security updates, without needing to first update the requirements.txt in this repo.

michaelwood commented:
Tests should pass after https://github.com/OpenDataServices/cove/pull/1336 is merged

R2ZER0 committed:
Loosen the requirements when the datagetter is used as a library, by switching setup.py to use requirements.in instead of requirements.txt.
Now that requirements.in specifies compatible ranges, this allows downstream library uses to upgrade to compatible sub-requirement versions, e.g. for security updates, without needing to first update the requirements.txt in this repo.
